kyuubi-common/src/main/scala/org/apache/kyuubi/service/TFrontendService.scala

```diff
@@ -66,12 +66,12 @@ abstract class TFrontendService(name: String)
   protected def stopServer(): Unit
 
   override def stop(): Unit = synchronized {
+    super.stop()
```

Comment:
Generally, the order of `super.start()` and `super.stop()` should be symmetrical

Reply:
what actually you are trying to say about this change？

Reply:
I think the lifecycle of FE should be: start server => register => serving... => unregister => stop server

Reply:
then what actually varies the order you present here,are we good to go?

Reply:
`super.start()` registers the FE before FE-server is ready, we should also move `super.start()` after `serverThread.start()`

Reply:
> Generally, the order of super.start() and super.stop() should be symmetrical

What I mean **symmetrical** is, normally the pattern should be one of

```
def start(): Unit = {
   super.start()
   self.start()
}
def stop(): Unit = {
   self.stop()
   super.stop()
}
```

```
def start(): Unit = {
   self.start()
   super.start()
}
def stop(): Unit = {
   super.stop()
   self.stop()
}
```

I'm just surprised that you changed it to
```
def start(): Unit = {
   self.start()
   super.start()
}
def stop(): Unit = {
   self.stop()
   super.stop()
}
```